index.html

```diff
@@ -13,11 +13,11 @@
 <body>
   <h1>Welcome to The Shade Generator!</h1>
 
-  <input type="text" class="red" value="Red Value"></input>
+  <input type="number"  max="255" min="0"class="red" placeholder="Red Value"></input>
```

Comment:
What are the max and min values for? Do they change how the JavaScript, or are they just to make the code easier to read?

Reply:
Oh. I see. They create the Up and Down buttons on the side. Great addition! Is there any way to make them match the website theme a little more? I feel that they stick out a little.